VirtoCommerce.CatalogModule.Web.Core/Model/Category.cs

```diff
@@ -35,13 +35,14 @@ public class Category : AuditableEntity, ISeoSupport, IHasOutlines
         /// The code.
```

Comment:
Are there significant changes? if no, need to revert.

Reply:
reverted